src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -339,6 +327,8 @@ class BaseOptionsSelector extends Component {
                 selectTextOnFocus
                 blurOnSubmit={Boolean(this.state.allOptions.length)}
                 spellCheck={false}
+                autoFocus={this.props.autoFocus}
+                shouldDelayFocus={this.props.shouldDelayFocus}
```

Comment:
Confirmed that this fixes [Keyboard opens multiple times when selecting members to request money](https://github.com/Expensify/App/pull/24041#issuecomment-1677294128).
Curious about the root cause though?